[2025-09-15T17:57:03.728+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-15T17:57:03.774+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-14T00:00:00+00:00 [queued]>
[2025-09-15T17:57:03.795+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-14T00:00:00+00:00 [queued]>
[2025-09-15T17:57:03.796+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-15T17:57:03.827+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task_2> on 2025-09-14 00:00:00+00:00
[2025-09-15T17:57:03.842+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mon_premier_dag', 'task_2', 'scheduled__2025-09-14T00:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/mon_premier_dag.py', '--cfg-path', '/tmp/tmppty8cbdv']
[2025-09-15T17:57:03.847+0000] {standard_task_runner.py:105} INFO - Job 152: Subtask task_2
[2025-09-15T17:57:03.850+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=245) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-15T17:57:03.851+0000] {standard_task_runner.py:72} INFO - Started process 246 to run task
[2025-09-15T17:57:03.927+0000] {task_command.py:467} INFO - Running <TaskInstance: mon_premier_dag.task_2 scheduled__2025-09-14T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-15T17:57:04.107+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mon_premier_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-09-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-14T00:00:00+00:00'
[2025-09-15T17:57:04.108+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-15T17:57:04.138+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-09-15T17:57:04.140+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "This is my first DAG!"']
[2025-09-15T17:57:04.166+0000] {subprocess.py:99} INFO - Output:
[2025-09-15T17:57:04.170+0000] {subprocess.py:106} INFO - This is my first DAG!
[2025-09-15T17:57:04.171+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-09-15T17:57:04.219+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-15T17:57:04.221+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mon_premier_dag, task_id=task_2, run_id=scheduled__2025-09-14T00:00:00+00:00, execution_date=20250914T000000, start_date=20250915T175703, end_date=20250915T175704
[2025-09-15T17:57:04.271+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-15T17:57:04.305+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-15T17:57:04.310+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
